src/spartan_tcp_handler.erl

```diff
@@ -39,7 +39,6 @@ callback_mode() ->
     state_functions.
 
 init([Ref, Socket, Transport, Opts]) ->
-    {ok, _} = timer:kill_after(?TIMEOUT * 5),
```

Comment:
There are `idle_timeout`, `query_timeout` & others. I don't know why someone added this line, It's enough to have everywhere appropriate timeouts.